### PR TITLE
[mod_finder]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_finder
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 2bd7924db0f0fa7f9a4f7011db65c0fd42da952a<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10583">issues.joomla.org/joomla-cms/10583</a>.</sub>

File: modules/mod_finder/mod_finder.xml [hunk 1]
```diff
@@ -25,110 +25,113 @@
...
-					default="1">
+					class="btn-group btn-group-yesno"
+					default="1"
+					>
 					<option value="1">JYES</option>
 					<option value="0">JNO</option>
 				</field>
+
 				<field
 					name="opensearch_title"
```
Comment (JoomliC) [inline modules/mod_finder/mod_finder.xml]: <code>/></code> to be on a new line for this field...

**Comment by JoomliC** [comment]
I have tested this item :red_circle: unsuccessfully on 2bd7924db0f0fa7f9a4f7011db65c0fd42da952a<br /><br />See: https://github.com/joomla/joomla-cms/pull/10583/files#r64242383<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10583">issues.joomla.org/joomla-cms/10583</a>.</sub>

File: modules/mod_finder/mod_finder.xml (hunk 1 above)
Comment (wojsmol) [inline modules/mod_finder/mod_finder.xml]: fixed

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 0d433677c4755117984682d7e13b1ccf85bbd84a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10583">issues.joomla.org/joomla-cms/10583</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 0d433677c4755117984682d7e13b1ccf85bbd84a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10583">issues.joomla.org/joomla-cms/10583</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10583">issues.joomla.org/joomla-cms/10583</a>.</sub>

**Pull request closed without merging**